addons/mail/static/src/discuss/call/common/dropdown_patch.js

```diff
@@ -0,0 +1,34 @@
+import { Dropdown } from "@web/core/dropdown/dropdown";
+import { patch } from "@web/core/utils/patch";
+
+Object.assign(Dropdown.props, {
+    openOnMouseEnter: {
+        type: Boolean,
+        optional: true,
+    },
+});
+Object.assign(Dropdown.defaultProps, {
+    openOnMouseEnter: true,
+});
+
+patch(Dropdown.prototype, {
+    handleMouseEnter() {
+        if (!this.props.openOnMouseEnter) {
+            return;
+        }
+        super.handleMouseEnter(...arguments);
+    },
+    handleClick(event) {
```

Comment:
`event` => `ev`

When I started at Odoo I remember we shouldn't name it `event` because a typo in code like missing naming the param would work in Chrome thanks to custom behavior in this browser but other browsers like Firefox wouldn't work.

I don't know if this problem still exists or not, but better keep the tradition to name it `ev` if we are unsure, as this is less prone to mistakes.